packages-ts/eqlabs-multisig-integration/test/Multisig.test.ts

```diff
@@ -0,0 +1,65 @@
+import { expect } from 'chai'
+import { starknet } from 'hardhat'
+import { StarknetContract, StarknetContractFactory, Account } from 'hardhat/types/runtime'
+import { getSelectorFromName } from 'starknet/dist/utils/hash'
+import { number } from 'starknet'
+
+describe('Multisig integration tests', function () {
+  this.timeout(300_000)
+
+  let account1: Account
+  let account2: Account
+  let account3: Account
+
+  let multisig: StarknetContract
+
+  before(async function () {
+    account1 = await starknet.deployAccount('OpenZeppelin')
+    account2 = await starknet.deployAccount('OpenZeppelin')
+    account3 = await starknet.deployAccount('OpenZeppelin')
+  })
+
+  it('Deploy contract', async () => {
+    let multisigFactory = await starknet.getContractFactory('Multisig')
+    multisig = await multisigFactory.deploy({
+      signers: [
+        number.toBN(account1.starknetContract.address),
+        number.toBN(account2.starknetContract.address),
+        number.toBN(account3.starknetContract.address),
+      ],
+      threshold: 2,
+    })
+
+    expect(multisig).to.be.ok
+  })
+
+  it('should submit & confirm transaction', async () => {
+    const nonce = 0
+    const selector = getSelectorFromName('get_signers')
+    const payload = {
+      to: multisig.address,
+      function_selector: selector,
+      calldata: [],
+      nonce,
+    }
```

Comment:
Please do a `set_treshold` here instead which is a more realistic example. 
Change it from 2 to 3, then execute, and verify the change in the end.